word.not.found.exception=Word {0} not exists.
collection.not.found.exception=Collection {0} not exists.
trial.not.found.exception=Trial {0} not exists.
user.not.found.exception=User {0} not exists.
result.not.found.exception=Result {0} not exists.

api.exception.default.message=Something bad happened. Try again later.
